Libraries/Opc.Ua.Server/Session/Session.cs

```diff
@@ -1174,7 +1174,9 @@ private void UpdateDiagnosticCounters(RequestType requestType, bool error, bool
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1823:AvoidUnusedPrivateFields")]
         private uint m_maxResponseMessageSize;
         private double m_maxRequestAge;
-        private int m_maxBrowseContinuationPoints;
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+        protected int m_maxBrowseContinuationPoints;
```

Comment:
This value normally comes from configuration, but I see no harm here in changing this at runtime, why not make it a protected property, then we also don´t need the warning here?